src/rns/hooks/utils.ts

```diff
@@ -1,5 +1,17 @@
 import { HookContext } from '@feathersjs/feathers'
 
+export function setSellerAddressParamHook (context: HookContext): HookContext {
```

Comment:
Sorry about this, my bad. Not sure if we have to have the `utils.ts` file anymore as it was mainly in place to de-duplicate code, but we can keep it for readability I guess.